Framework/Core/include/Framework/AnalysisHelpers.h

```diff
@@ -20,6 +20,9 @@
 #include "Framework/OutputObjHeader.h"
 #include "Framework/StringHelpers.h"
 #include "Framework/Output.h"
+#include "Framework/IndexBuilderHelpers.h"
+#include <arrow/table.h>
+#include <arrow/util/key_value_metadata.h>
```

Comment:
Can't we hide these in the .cxx via some forward declaration?

Reply:
Yes, I think I can move the whole table creation into .cxx. I will update this later today.